ontology/eco.md

```diff
@@ -1,7 +1,11 @@
 ---
 layout: ontology_detail
 id: eco
-title: Evidence ontology
+title: Evidence and Conclusion Ontology
+browsers:
+- title: ECO Browser
+  label: ECO
+  url: https://www.evidenceontology.org/
```

Comment:
@nsuvarnaiari Thank you for fixing the title and adding a link to the browser.

It looks like the browser URL is just the ECO landing page.

I would recommend `https://www.evidenceontology.org/browse/` as the browser URL instead:

```suggestion
  url: https://www.evidenceontology.org/browse
```